### PR TITLE
Fastest almost-solution to this problem

### PR DESCRIPTION
**Pull request by DeromirNeves**

### CODE REVIEW

File: PRIME1/solution.ex [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  defp _is_divided_by?(i, n) when rem(i, 6) == 1, do: _is_divided_by? i + 4, n
+  defp _is_divided_by?(i, n) when rem(i, 6) == 5, do: _is_divided_by? i + 2, n
+
+  defp _has_divisors?(n) do
+    _is_divided_by? 2, n
+  end
+
+  def find(l) do
+    [m, n] = IOUtils.parse_input_line l
+    for i <- m..n do
```
Comment (RenanBardy) [inline PRIME1/solution.ex:37]: What do you think about change this `for` to a `Enum.each` function

```elixir
Enum.each(m..n, &_print_prime(&1))
```

**Review by RenanBardy — COMMENTED** [review]
I've just added one suggestion about the `for` loop

**Comment by RenanBardy** [comment]
There is the out file too

File: PRIME1/solution.ex (hunk 1 above)
Comment (DeromirNeves) [reply PRIME1/solution.ex:37]: You can try that and check if the execution time is reduced. I'm not sure it will.

**Comment by DeromirNeves** [comment]
I'll drop this pull request, because it doesn't solve the problem

**Pull request closed without merging**